src/components/StatusChip.js

```diff
@@ -2,22 +2,19 @@ import React from 'react';
 import Chip from '@material-ui/core/Chip';
 import { CheckCircle, Cancel } from '@material-ui/icons';
 
-class StatusChip extends React.PureComponent {
-  render() {
-    const { type, data } = this.props;
-    switch (type) {
-      case 'certified':
-        return (
-          <Chip
-            avatar={<CheckCircle />}
-            label={`Gap Certified since ${data.certificationDate}`}
-          />
-        );
-      case 'notCertified':
-        return <Chip avatar={<Cancel />} label="Not GAP Certified" />;
-      default:
-        return null;
-    }
+export default function StatusChip(props) {
+  const { type, data } = props;
+  switch (type) {
```

Comment:
Converted all share components to be functional components for easier readability.